### ISSUE TITLE
Categorize the Add Accessory List

### ISSUE THREAD

**Issue by BobbySlope** [hoobs-org/HOOBS#163]
Maybe show all accessories in edit rooms and by hiding them make them transparent or crosslined. The users dont get it, that they need to scroll down to the checkbox list.

**Comment by mkellsy** [comment]
To unhide you just add the Accessory to a room. It's the same as adding an accessory to a room. Hiding an accessory removes it from a room, so if I were to show them they would always be Unassigned. Then to move the accessory to a room, they would have to add it to a room in the same fashion as unhiding.

I don't know how much easier I can make it.

**Comment by mkellsy** [comment]
I made the add accessory list to show that an accessory is hidden and it also categorizes the accessories in rooms. All unassigned or hidden accessories go on the top of the list. This is the Google Home model.

The unassigned accessories show first then the hidden accessories. I also added a (hidden) label on the hidden accessories.

<img width="477" alt="Screen Shot 2019-09-26 at 10 05 04 AM" src="https://user-images.githubusercontent.com/9777975/65705344-7dd53200-e045-11e9-94e5-a37c2ac022c6.png">

Done in 3.0.0-beta.58
